modules/nf-core/rgi/main/meta.yml

```diff
@@ -22,6 +22,15 @@ input:
       type: file
       description: Nucleotide or protein sequences in FASTA format
       pattern: "*.{fasta,fasta.gz,fa,fa.gz,fna,fna.gz,faa,faa.gz}"
+  - card:
+      type: directory
+      description: Directory containing the CARD
+      pattern: "*/"
+  - wildcard:
+      type: directory
+      description: Directory containing the WildCARD database (optional)
```

Comment:
Maybe specify that the structure must unaltered from being downloaded, i.e. XYZ file should be in XYZ position or w/e